phobos/blender/model/inertia.py

```diff
@@ -201,7 +201,13 @@ def combine_com_3x3(objects):
     for obj in objects:
         combined_com = combined_com + obj.matrix_local.translation * obj['mass']
         combined_mass += obj['mass']
-    combined_com = combined_com / combined_mass
+    if combined_mass != 0:
+        combined_com = combined_com / combined_mass
+    else:
+        combined_com = mathutils.Vector((0.0,) * 3)
+        for obj in objects:
+            combined_com = combined_com + obj.matrix_local.translation
+        log(message=f"Inertial {objects[0].name} exported with mass 0")
```

Comment:
The way you put the else part the combined_com will be somewhere outside the objects. If there is now mass we should take the center of volumes. Therefore the obj.matrix_local.translation should be weighted with the volume off the obj (convex hull if not watertight).